migrate/page-index/page-index.js

```diff
@@ -15,6 +15,11 @@ module.exports = [
         href: "live-migration",
         excerpt: "Migrate a large database with low downtime",
       },
+      {
+        title: "Live sync",
```

Comment:
Same "Postgres Livesync", for naming consistency can be name as one word livesync?

Reply:
As above. 
Livesync is now a proper noun for this feature. Maybe we can change the name of the image to livesync as well?